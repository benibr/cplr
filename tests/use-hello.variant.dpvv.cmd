../bin/cplr 'puts("Hello.")' -dpvv
