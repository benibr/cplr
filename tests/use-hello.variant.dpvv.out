Preparation phase
Generation phase
Generating code
Generating section statements
================================================================================
     1	/* main */
     2	#line 148 "internal"
     3	int main(int argc, char **argv) {
     4	    int ret = 0;
     5	/* statements */
     6	#line 1 "statements_0"
     7	    puts("Hello.");
     8	/* done */
     9	#line 163 "internal"
    10	    return ret;
    11	}
================================================================================
Generated: 193 bytes code, 193 bytes dump
Compilation phase
Compiler condition:
  statements_0:1: warning: implicit declaration of function 'puts'
Compilation succeeded
Execution phase
Execution finished (ret=0).
Writing history
Hello.
